recipes/nf-test/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+BINARY_HOME=$PREFIX/bin
+PACKAGE_HOME=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+
+mkdir -p $PREFIX/bin
+mkdir -p $PACKAGE_HOME
+
+sed -i "s|APP_HOME=.*|APP_HOME=\"${PACKAGE_HOME}\"|g" nf-test
+
+cp nf-test* $PACKAGE_HOME
```

Comment:
@matthdsm looks like there are multiple files copied over; are there any other executables that need to be symlinked as well?

Reply:
nope, there's only a jar file and the execution script!